src/models/balenaos-contract.ts

```diff
@@ -12,14 +12,14 @@ export const BalenaOS: Contract = {
 		image: [`{{#each deviceType.partials.instructions}}{{{this}}} {{/each}}`],
 		internalFlash: [
 			`{{#each deviceType.partials.connectDevice}}{{{this}}} {{/each}}`,
-			`Write the {{name}} file you downloaded to the {{deviceType.name}}. We recommend using <a href="http://www.etcher.io/">Etcher</a>.`,
+			`Write the {{name}} file you downloaded to the {{deviceType.name}}. We recommend using <a href="https://www.etcher.balena.io/">Etcher</a>.`,
```

Comment:
This url doesn't exist for me, so I guess the correct is:
```suggestion
			`Write the {{name}} file you downloaded to the {{deviceType.name}}. We recommend using <a href="https://etcher.balena.io/">Etcher</a>.`,
```
or
```suggestion
			`Write the {{name}} file you downloaded to the {{deviceType.name}}. We recommend using <a href="https://balena.io/etcher">Etcher</a>.`,
```

Applies in all changes.

Reply:
Yeah, one problem at a time. It seems there is no CNAME entry for www on this URL. Will fix that in the backend till then.